I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1916, in script
    image char_nom = char_nom
  File "game/script.rpy", line 1916, in <module>
    image char_nom = char_nom
NameError: name 'char_nom' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1916, in script
    image char_nom = char_nom
  File "C:\renpy-7.2.2-sdk\renpy\ast.py", line 999, in execute
    img = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\renpy-7.2.2-sdk\renpy\python.py", line 1954, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 1916, in <module>
    image char_nom = char_nom
NameError: name 'char_nom' is not defined

Windows-8-6.2.9200
Ren'Py 7.2.2.491
버드나무 꽃 만개할 그날까지 1.1
Thu Jun 20 17:09:19 2019
